src/scripts/main.js

```diff
@@ -1 +1,79 @@
 'use strict';
+
+function createPromise1(button, message) {
+  return new Promise((resolve, reject) => {
+    button.addEventListener('click', () => {
+      resolve();
+
+      button.removeEventListener('click', () => {
```

Comment:
it will not work like that, you should save function in variable and add/remove this variable